custom_components/grocy/__init__.py

```diff
@@ -5,14 +5,15 @@
 import logging
 import os
 from datetime import timedelta
+import asyncio
```

Comment:
I guess this could become a local import as it's only used in `handle_execute_chore` atm.

Reply:
Not sure... In doc exemple it seems to be on top : https://developers.home-assistant.io/docs/en/asyncio_working_with_async.html#calling-async-functions-from-threads

Let me know

Reply:
should be fine then